public/popup.css

```diff
@@ -6,6 +6,10 @@
   --sb-green-bg: #077B27;
 }
 
+html {
+  color-scheme: dark;
```

Comment:
I think this is going to make the YouTube page have dark scrollbars too

Reply:
If you add the property `html {color-scheme: dark; }` to content.css or popup.css, then when loading YouTube you can see a dark scroll bar for a split second, but then YT changes it to its own custom scrollbar, and I couldn't find the use of the standard scroll bar anywhere on YT.

Reply:
Ah, that makes sense.